cdap-app-fabric/src/main/java/co/cask/cdap/adapter/AdapterType.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.adapter;
+
+/**
+ * Defines the type of Adapters supported by the system.
+ */
+public enum AdapterType {
+
+  /**
+   * The adapter will be executed via batch jobs and convert data in stream to Avro.
+   */
+  BATCH_STREAM_TO_AVRO,
```

Comment:
Does `Avro` part of the type? It seems contradict to when you mentioned above that an Adapter can convert into multiple sink with different format.